src/arkit.cs

```diff
@@ -509,6 +518,106 @@ interface ARSCNDebugOptions {
 		[Field ("ARSCNDebugOptionShowFeaturePoints")]
 		SCNDebugOptions ShowFeaturePoints { get; }
 	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[Protocol]
+	interface ARTrackable {
+		[Abstract]
+		[Export ("isTracked")]
+		bool IsTracked { get; }
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARConfiguration))]
+	interface ARFaceTrackingConfiguration {}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARAnchor))]
+	interface ARFaceAnchor : ARTrackable {
+		[Export ("geometry")]
+		ARFaceGeometry Geometry { get; }
+
+		/* Will come in next PR
+		// @property (readonly, nonatomic) NSDictionary<ARBlendShapeLocation,NSNumber *> * _Nonnull blendShapes;
+		[Export ("blendShapes")]
+		NSDictionary<NSString, NSNumber> BlendShapes { get; }
+		*/
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface ARFaceGeometry : NSCopying {
+		/* Will come in next PR
+		[Export ("initWithBlendShapes:")]
+		IntPtr Constructor (NSDictionary<NSString, NSNumber> blendShapes);
+		*/
+
+		[Export ("vertexCount")]
+		nuint VertexCount { get; }
+
+		// @property (readonly, nonatomic) const vector_float3 * _Nonnull vertices;
+		[Export ("vertices")]
+		Vector3 Vertices {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("textureCoordinateCount")]
+		nuint TextureCoordinateCount { get; }
+
+		[Export ("textureCoordinates")]
+		Vector2 TextureCoordinates {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("triangleCount")]
+		nuint TriangleCount { get; }
+
+		[Export ("triangleIndices")]
+		short TriangleIndices { get; }
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(SCNGeometry))]
+	[DisableDefaultCtor]
+	interface ARSCNFaceGeometry {
+		[Static]
+		[Export ("faceGeometryWithDevice:")]
+		[return: NullAllowed]
+		ARSCNFaceGeometry CreateFaceGeometry (IMTLDevice device);
+
+		[Static]
+		[Export ("faceGeometryWithDevice:fillMesh:")]
+		[return: NullAllowed]
+		ARSCNFaceGeometry CreateFaceGeometry (IMTLDevice device, bool fillMesh);
+
+		[Export ("updateFromFaceGeometry:")]
+		void Update (ARFaceGeometry faceGeometry);
+	}
+
+	[iOS (11,0)]
+	[NoWatch, NoTV, NoMac]
+	[BaseType (typeof(ARLightEstimate))]
```

Comment:
base type has a `[DisableDefaultCtor]` so the subclass should too

Reply:
You're right subclass doesn't have `init`